pkg/controller/storagecluster/initialization_reconciler.go

```diff
@@ -205,24 +208,36 @@ func (r *ReconcileStorageCluster) ensureExternalStorageClusterResources(instance
 				// Setting the PoolName for RBD StorageClass
 				sc = scs[1]
 			} else if d.Name == cephRgwStorageClassName {
+				rgwEndpoint := d.Data[externalCephRgwEndpointKey]
+				extCephObjectStores, err = r.newExternalCephObjectStoreInstances(instance, rgwEndpoint, reqLogger)
+				if err != nil {
+					return err
+				}
+				// rgw-endpoint is no longer needed in the 'd.Data' dictionary,
+				// and can be deleted
+				delete(d.Data, externalCephRgwEndpointKey)
 				// Set the external rgw endpoint variable for later use on the Noobaa CR (as a label)
 				// Replace the colon with an underscore, otherwise the label will be invalid
-				externalRgwEndpointReplaceColon := strings.Replace(d.Data[externalCephRgwEndpointKey], ":", "_", -1)
+				externalRgwEndpointReplaceColon := strings.Replace(rgwEndpoint, ":", "_", -1)
```

Comment:
I need to check with @nb-ohad, for changes in Noobaa but this might need to go away as well.